tests/test_project.py

```diff
@@ -3066,5 +3068,185 @@ def helper_test_export_project(project):
+            ],
+        )
+        assert False
+    except SherlockCreateCCAFromModelingRegionError as e:
+        assert str(e) == "Create CCA from modeling region error: CCA name is missing for CCA 0."
+
+    try:
+        project.create_cca_from_modeling_region(
+            "Test",
+            [
+                {
+                    "cca_name": "",
+                    "modeling_region_id": "MR1",
+                    "description": "Test",
+                    "default_solder_type": "SAC305",
+                    "default_stencil_thickness": 10,
+                    "default_stencil_thickness_units": "mm",
+                    "default_part_temp_rise": 20,
+                    "default_part_temp_rise_units": "C",
+                    "guess_part_properties": False,
+                    "generate_image_layers": False,
+                },
+            ],
+        )
+        assert False
+    except SherlockCreateCCAFromModelingRegionError as e:
+        assert str(e) == "Create CCA from modeling region error: CCA name is invalid for CCA 0."
+
+    try:
+        project.create_cca_from_modeling_region(
+            "Test",
+            [
+                {
+                    "cca_name": "Main Board",
+                    "description": "Test",
+                    "default_solder_type": "SAC305",
+                    "default_stencil_thickness": 10,
+                    "default_stencil_thickness_units": "mm",
+                    "default_part_temp_rise": 20,
+                    "default_part_temp_rise_units": "C",
+                    "guess_part_properties": False,
+                    "generate_image_layers": False,
+                },
+            ],
+        )
+        assert False
+    except SherlockCreateCCAFromModelingRegionError as e:
+        assert (
+            str(e) == "Create CCA from modeling region error: Modeling Region ID"
+            " is missing for CCA 0."
+        )
+
+    try:
+        project.create_cca_from_modeling_region(
+            "Test",
+            [
+                {
+                    "cca_name": "Card",
+                    "modeling_region_id": "",
+                    "description": "Test",
+                    "default_solder_type": "SAC305",
+                    "default_stencil_thickness": 10,
+                    "default_stencil_thickness_units": "mm",
+                    "default_part_temp_rise": 20,
+                    "default_part_temp_rise_units": "C",
+                    "guess_part_properties": False,
+                    "generate_image_layers": False,
+                },
+            ],
+        )
+        assert False
+    except SherlockCreateCCAFromModelingRegionError as e:
+        assert (
+            str(e) == "Create CCA from modeling region error: Modeling Region ID"
+            " is invalid for CCA 0."
+        )
+
+    if not project._is_connection_up():
+        return
+
+    try:
+        project.create_cca_from_modeling_region(
+            "Test",
```

Comment:
Change the project name to be "Tutorial Project".
Technically, this test will give the same results if there isn't a project named "Test". However, the wording in the call to `pytest.fail()`, indicates that the intent is to check a failure caused by parameter validation in Sherlock, rather than no project with a valid project name.